playbooks/wordpress-docker.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Deploy a WordPress instance using docker
+  hosts: wordpress-docker
+  become: true
+  pre_tasks:
+      # Required by the consul role.
+    - name: Ensure that unzip is installed
+      apt:
+        name: unzip
+        state: present
+        update_cache: yes
```

Comment:
Shouldn't it live in `consul` role than?

Reply:
@ahmedbilal, this could live in the `consul` role, yes. But I was trying to avoid making changes to any existing roles. It works okay on all other playbooks because `unzip` is automatically installed by the `common-server` role that is included in all the other playbooks.

I will update the PR to add this to the `consul` role itself since it is an explicit dependency.